src/frontend/specification/ParameterTable.jl

```diff
@@ -276,9 +359,207 @@ Write hessian standard errors computed for `sem_fit` to the `:se` column of `par
 """
 function update_se_hessian!(
     partable::AbstractParameterTable,
-    sem_fit::SemFit;
+    fit::SemFit;
     hessian = :finitediff,
 )
-    se = se_hessian(sem_fit; hessian = hessian)
-    return update_partable!(partable, sem_fit, se, :se)
+    se = se_hessian(fit; hessian = hessian)
+    return update_partable!(partable, :se, params(fit), se)
+end
+
+"""
+    param_values!(out::AbstractVector, partable::ParameterTable,
+                  col::Symbol = :estimate)
+
+Extract parameter values from the `col` column of `partable`
+into the `out` vector.
+
+The `out` vector should be of `nparams(partable)` length.
+The *i*-th element of the `out` vector will contain the
+value of the *i*-th parameter from `params(partable)`.
+
+Note that the function combines the duplicate occurences of the
+same parameter in `partable` and will raise an error if the
+values do not match.
+"""
+function param_values!(
+    out::AbstractVector,
+    partable::ParameterTable,
+    col::Symbol = :estimate,
+)
+    (length(out) == nparams(partable)) || throw(
+        DimensionMismatch(
+            "The length of parameter values vector ($(length(out))) does not match the number of parameters ($(nparams(partable)))",
+        ),
+    )
+    param_index = Dict(param => i for (i, param) in enumerate(params(partable)))
+    param_values_col = partable.columns[col]
+    for (i, param) in enumerate(partable.columns[:param])
+        (param == :const) && continue
+        param_ind = get(param_index, param, nothing)
+        @assert !isnothing(param_ind) "Parameter table contains unregistered parameter :$param at row #$i"
+        val = param_values_col[i]
+        if !isnan(out[param_ind])
+            @assert out[param_ind] ≈ val atol = 1E-10 "Parameter :$param value at row #$i ($val) differs from the earlier encountered value ($(out[param_ind]))"
```

Comment:
If we wan't to keep the approximate comparison, we would have to write 
```suggestion
            @assert isapprox(out[param_ind], val, atol = 1e-10) "Parameter :$param value at row #$i ($val) differs from the earlier encountered value ($(out[param_ind]))"
```
because otherwise, it trows `AssertionError: 1.0e-10`, omitting the message.